mobile/src/main/java/com/abrenoch/hyperiongrabber/mobile/SettingsActivity.java

```diff
@@ -51,12 +49,12 @@ private static void bindPreferenceSummaryToValue(Preference preference) {
         // Set the listener to watch for value changes.
         preference.setOnPreferenceChangeListener(sBindPreferenceSummaryToValueListener);
 
+        Preferences prefs = new Preferences(preference.getContext());
+
         // Trigger the listener immediately with the preference's
         // current value.
         sBindPreferenceSummaryToValueListener.onPreferenceChange(preference,
-                PreferenceManager
-                        .getDefaultSharedPreferences(preference.getContext())
-                        .getString(preference.getKey(), ""));
+                prefs.getString(preference.getKey(), ""));
```

Comment:
I would slightly prefer this change to be reversed and the corresponding method to be removed from `Preferences.kt`. The reason being 
- The net result will be exactly the same
- It introduces a way to work with strings as keys in Preferences.kt, which counters it's purpose
- It introduces asymmetry: Preferences.kt is used for reading values, but SharedPreferences is used for writing values

Reply:
Sounds alright to me - agree with the reasoning too!